Detectors/TOF/workflow/src/TOFClusterizerSpec.cxx

```diff
@@ -75,7 +75,8 @@ class TOFDPLClustererTask
     mTimer.Start(false);
     // get digit data
     auto digits = pc.inputs().get<gsl::span<o2::tof::Digit>>("tofdigits");
-    auto row = pc.inputs().get<std::vector<o2::tof::ReadoutWindowData>*>("readoutwin");
+    auto rowp = pc.inputs().get<std::vector<o2::tof::ReadoutWindowData>>("readoutwin");
+    auto row = &rowp;
```

Comment:
`ReadoutWindowData` is POD, why do you use vector instead of the span?

Reply:
There is not a specific reason... moved to gsl::span